packages/build/src/html-transform.ts

```diff
@@ -152,8 +156,21 @@ export function htmlTransform(
   }
 
   if (options.injectBabelHelpers) {
+    let helpers;
+    switch (options.injectBabelHelpers) {
+      case 'full':
+        helpers = getMinfiedBabelHelpersFull();
+        break;
+      case 'amd':
+        helpers = getMinifiedBabelHelpersAmd();
+        break;
+      default:
+        throw new Error(
```

Comment:
Use the type checker for completeness checking:

```typescript
const never: never = options.injectBabelHelpers;
throw new Error(`Unknown injectBabelHelpers value: ${never}`);
```

Reply:
Done.